init-tools.sh

```diff
@@ -121,7 +121,13 @@ if [ ! -e $__DOTNET_PATH ]; then
     __DOTNET_LOCATION="https://dotnetcli.azureedge.net/dotnet/Sdk/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.tar.gz"
 
     install_dotnet_cli() {
-        echo "Installing '${__DOTNET_LOCATION}' to '$__DOTNET_PATH/dotnet.tar'" >> "$__init_tools_log"
+        if [[ ! "$DotNetBootstrapCliTarPath" = "" ]]; then
+            echo "Running 'tar -xf $DotNetBootstrapCliTarPath' in '$__DOTNET_PATH'"
+            cd $__DOTNET_PATH
+            tar -xf $DotNetBootstrapCliTarPath
+            return
+        fi
+        echo "Installing '${__DOTNET_LOCATION}' to '$__DOTNET_PATH/dotnet.tar'"
```

Comment:
Removed the `>> "$__init_tools_log"` for consistency and because it is already handled in the `install_dotnet_cli()` call below in line 135 / 141